fractal-puppy/Engine
# Multiple improvements of the pathing and mouse following

To test this feature:
Personal recomendation: delete the objects that generate particles (in order to improve performance)
- load the graveyard scene
- press play
- see that now player follows mouse even when mouse passes through the position of the player
- see that if clicking outside the map. If not too far, the player goes to the edge of the map.
- see that Dash also does that
